### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by crispen-smith**
Added additional clarity around module purpose.

### CODE REVIEW

**Comment by dougwilson** [comment]
Thank you!

File: README.md [hunk 1]
```diff
@@ -104,6 +104,8 @@ var server = http.createServer(function onRequest(req, res) {
 server.listen(3000);
 ```
 
+
+For sites using a more cross-browser solution this module will only provide the default, the serve-static middleware may be a better solution for any additional icons.
```
Comment (dougwilson) [inline README.md]: I'm not sure what the sentence really means. Can you expand it? This module provides the _only_ cross-browser solution. It woks as far back as IE 5.5 and across all browsers and all operating systems.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [![Test Coverage][coveralls-image]][coveralls-url]
 [![Gittip][gittip-image]][gittip-url]
 
-Node.js middleware for serving a favicon.
+Node.js middleware for serving the browser default favicon.
```
Comment (dougwilson) [inline README.md]: This file is not limited to only web browsers, so this addition doesn't really make sense; as an example, using most RSS readers, they will show the `favicon.ico` file, but aren't considered to be browsers. They also show up on shortcuts to sites on at least Windows desktops.

**Comment by dougwilson** [comment]
Perhaps just linking to https://en.wikipedia.org/wiki/Favicon may help people?

**Comment by crispen-smith** [comment]
I think that linking it is a really good idea but I'm sure I can make the phrasing of the highlights more clear let me take another crack at it.

**Comment by dougwilson** [comment]
I'm in no rush :)

**Comment by adius** [comment]
This should be merged! Just stumbled upon the same problem as described in #22.
(…but please fix the typos first ;-) )

**Comment by dougwilson** [comment]
I did not realize the PR had been updated.

**Comment by crispen-smith** [comment]
Reviewed again and made a few minor additional changes.

**Comment by dougwilson** [comment]
Looks good. Will merge :) Sorry for the delay; I just didn't realize it was updated before :O

**Comment by crispen-smith** [comment]
Just glad I was able to help.

**Pull request closed without merging**